pkg/sink/codec/simple/message.json

```diff
@@ -28,8 +28,10 @@
       },
       {
         "name": "elements",
-        "type": "array",
-        "items": "string"
+        "type": {
```

Comment:
Please describe the reasons for such changes in the pr body.

Reply:
It's changed to follow the official convention.